azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -1011,6 +1020,76 @@ private void ajaxExecuteFlow(final HttpServletRequest req,
     ret.put("execid", exflow.getExecutionId());
   }
 
+  /**
+   * This method is used to update the property. propType: Is the umbrella for properties for which
+   * values need to be updated subType: Actual property for which values need to be updated val:
+   * value to be updated
+   * <p>
+   * Example: propType=containerDispatch&subType=updateAllowList&val=spark,java
+   */
+  private void ajaxUpdateProperty(final HttpServletRequest req,
+      final HttpServletResponse resp, final HashMap<String, Object> ret, final User user)
+      throws ServletException {
+    try {
+      if (!HttpRequestUtils.hasPermission(this.userManager, user, Type.ADMIN)) {
+        ret.put("error", String.format("User %s doesn't have ADMIN permission for updating "
+            + "property", user));
+      }
+      String propType = getParam(req, "propType");
+      if (propType.equals("containerDispatch")) {
+        updateContainerDispatchProps(req, ret);
```

Comment:
Can you please add a check to see if containerization enabled only then this method should be called. With that, can you please change method name to updatePropsForContainerizationDispatch ?

Reply:
done